media-video/qmplay2/qmplay2-18.12.26.ebuild

```diff
@@ -0,0 +1,125 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PLOCALES="de es fr hu pl ru uk zh"
+
+inherit cmake-utils l10n xdg
+
+MY_PN="QMPlay2"
+
+DESCRIPTION="Qt-based video player, which can play all formats and stream"
+HOMEPAGE="https://github.com/zaps166/${MY_PN}"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/zaps166/${MY_PN}.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/zaps166/${MY_PN}/releases/download/${PV}/${MY_PN}-src-${PV}.tar.xz"
+	KEYWORDS="~amd64 ~x86"
+	S=${WORKDIR}/${MY_PN}-src-${PV}
+fi
+
+LICENSE="LGPL-3"
+SLOT="0"
+
+CHIPTUNE=( gme sid )
+CORE=( avresample libass )
+EXTENSIONS=( animeodcinki myfreemp3 lastfm tekstowo wbijam )
+FFMPEG=( avdevice vaapi vdpau )
+GUI=( jemalloc taglib )
+MODULES=( alsa audiofilters cuvid extensions ffmpeg inputs modplug portaudio pulseaudio videofilters visualizations )
+
+IUSE="${CHIPTUNE[@]} ${CORE[@]} ${EXTENSIONS[@]} ${FFMPEG[@]} ${GUI[@]} ${MODULES[@]} cdio dbus libav mpris notifications opengl xv"
+
+REQUIRED_USE="
+	animeodcinki? ( extensions )
+	avdevice? ( ffmpeg )
+	myfreemp3? ( extensions )
+	lastfm? ( extensions )
+	mpris? ( extensions )
+	tekstowo? ( extensions )
+	vaapi? ( ffmpeg )
+	vdpau? ( ffmpeg )
+	wbijam? ( extensions )
+"
+
+RDEPEND="
+	avresample? ( >=media-video/ffmpeg-4.1 )
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtx11extras:5
+	dbus? ( dev-qt/qtdbus:5 )
+	gme? ( media-libs/game-music-emu )
+	cdio? ( dev-libs/libcdio[cddb] )
+	jemalloc? ( dev-libs/jemalloc )
+	libass? ( media-libs/libass )
+	libav? ( media-video/libav:= )
+	!libav? ( >=media-video/ffmpeg-3.1:= )
+	mpris? ( dev-qt/qtdbus:5 )
+	portaudio? ( media-libs/portaudio )
+	pulseaudio? ( media-sound/pulseaudio )
```

Comment:
most of flags you provided here do not affect configure stage wither way, we do not accept RDEPEND only use flags, please remover all the flags that do not match `src_configure()`

Reply:
pulseaudio and other flags located above in to the MODULES=() array and checkable by "for" construction with in src_configure()

do i need to change something? Thank you!